services/brig/src/Brig/API/Internal.hs

```diff
@@ -307,35 +307,11 @@ createUserNoVerify uData = do
   let uid = userId usr
   let eac = createdEmailActivation result
   let pac = createdPhoneActivation result
-
-  if isNewUserInvitedViaScim uData
```

Comment:
@fisx Removed sending activation email here. Should this be added somewhere else now?